app/controllers/jobs_controller.rb

```diff
@@ -1,11 +1,14 @@
+# frozen_string_literal: true
+
 class JobsController < ApplicationController
-  before_action :authenticate_user!, only: [:new, :create, :update, :edit, :my_jobs]
+  before_action :authenticate_user!, only: %i[new create update edit my_jobs]
 
   def index
-    @categories = JobCategory.all
+    @categories = Job.search(params[:old_vacancies])
   end
 
   def my_jobs
+    # method empty
```

Comment:
poderia remover a linha e o comentário.
```suggestion
    def my_jobs; end
```